src/appshell/view/preferences/folderspreferencesmodel.h

```diff
@@ -67,18 +73,28 @@ class FoldersPreferencesModel : public QAbstractListModel, public async::Asyncab
         Images
     };
 
-    struct FolderInfo {
-        FolderType type = FolderType::Undefined;
+    enum class CategoryValueType {
+        Directory,
+        MultiDirectories
+    };
+
+    struct CategoryInfo {
```

Comment:
I would suggest using a term that will be more like the old one (FolderInfo or even FoldersInfo)